pt/endpoints/user/resource.py

```diff
@@ -0,0 +1,90 @@
+from flask import jsonify, request, make_response
+from flask_restful import Resource
+from werkzeug.security import generate_password_hash, check_password_hash
+from utils.logging import logging
+from utils.oauth import auth, g
+from .model import User
+
+
+class UserResource(Resource):
+    # pylint: disable=R0201
+    @auth.login_required
+    def get(self):
+        logging.info(
+            "[Get User Request]\nUser Account:%s\nUUID:%s" % (g.account, g.uuid)
+        )
+        user = User.query.filter_by(uuid=g.uuid).first()
+        response = jsonify(
+            {
+                "username": user.username,
+                "avatar": user.avatar,
+                "balance": user.balance,
+                "address": user.address,
+                "eth_address": user.eth_address,
+            }
+        )
+        response.status_code = 200
```

Comment:
If you do not set the status_code 200, will it return other status code?

Reply:
Fix in db7106c.
Thanks for your information!